### DISCUSSION TITLE
ManimCE plug-in for VS Code

### DISCUSSION THREAD

**Discussion by pritam-mukherjee** [ManimCommunity/manim#2038] [Suggestions and Proposals]
Hey, it would be a life-saver if VS Code had a plug-in for ManimCE. Would make the coding experience for beginners easier and more friendly (as a beginner myself). If you have other ideas about this then please, I'd love to hear them. Thanks.

**Comment by behackl** [comment]
What sort of features would you expect from a VS Code plugin?

One particularly annoying problem is that VS Code currently (at least the last time I checked) does not support opening and playing videos in a separate tab. GIFs yes, but no MP4 etc.
